### DISCUSSION TITLE
Timezone not working

### DISCUSSION THREAD

**Discussion by Krokof4nt** [bbtsoftware/docker-mssql-backup#50] [Q&A] [answered]
I can't set timezone.
Example, TZ is set to Europe/Zurich in the variables but no effect.
Tested tzselect in docker terminal but it says 'time zone files are not set up correctly'.
So now the clock is one hour behind.
Any solution?

**Comment by Krokof4nt** [comment]
I set TZ=UTC-1 , then it showed correct time for my timezone (Europe/Stockholm)
Reason is to get correct timestamp on the backup-files, at first I set the cron to be an hour earlier.
Lets see when daylight saving is coming if it still works.

**Comment by stefangweichinger** [comment]
I also see this behavior. Although even with `TZ=UTC-1` I get the errors with `tzselect`. Manually changing it (for a test):

```
root@9a48c31dcab1:/# export TZ=Europe/Vienna
root@9a48c31dcab1:/# tzselect 
/usr/bin/tzselect: line 180: /usr/share/zoneinfo/iso3166.tab: No such file or directory
/usr/bin/tzselect: time zone files are not set up correctly
```

**Comment by Krokof4nt** [comment]
Daylight saving in sweden is not working with UTC-1, has to change to UTC-2 when daylight saving is active.
Is it possible to have this automatic adjusted?

**Comment by Krokof4nt** [comment, marked as answer]
I solved it.
In a terminal, apt-get install tzdata
some error occured but then it's possible to use Europe/Stockholm as TZ zone
